addons/containerd/template/script.sh

```diff
@@ -177,6 +181,20 @@ function find_common_versions() {
             add_supported_os_to_manifest_file $version "rhel-8" "Dockerfile.centos8"
         fi
 
+        if ! contains "$version" ${RHEL9_VERSIONS[*]}; then
+            echo "RHEL 9 lacks version $version"
+            add_unsupported_os_to_preflight_file "$version" "centos" "9"
+            add_unsupported_os_to_preflight_file "$version" "rhel" "9"
+            add_unsupported_os_to_preflight_file "$version" "ol" "9"
+            add_unsupported_os_to_preflight_file "$version" "rocky" "9"
```

Comment:
Should we also list these OSes for centos 8?
(and add rhel to centos 7?)

Reply:
added